components/org.wso2.carbon.identity.conditional.auth.functions.elk/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/elk/internal/ElasticFunctionsServiceComponent.java

```diff
@@ -160,4 +160,27 @@ protected void unsetIdentityGovernanceService(IdentityGovernanceService identity
         }
         // Do nothing.
     }
+
+    @Reference(
+            name = "server.configuration.service",
+            service = ServerConfigurationService.class,
+            cardinality = ReferenceCardinality.MANDATORY,
+            policy = ReferencePolicy.DYNAMIC,
+            unbind = "unsetServerConfigurationService"
+    )
+    protected void setServerConfigurationService(ServerConfigurationService serverConfigurationService) {
+
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("Setting the serverConfigurationService");
```

Comment:
Lets add a fullstop at the end. Check other debug logs too.

Reply:
Fixed with https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/110